components/profile-controller/controllers/monitoring.go

```diff
@@ -11,11 +11,12 @@ import (
 const PROFILE = "profile_controller"
 const COMPONENT = "component"
 const KIND = "kind"
+
 // User that make the request
 const REQUSER = "user"
 const ACTION = "action"
 const PATH = "path"
-const SEVERITY  = "severity"
+const SEVERITY = "severity"
```

Comment:
I would recommend just adding minimum change that we need. Style change may introduce more conflict adjustment in the future

Reply:
this is the result of `go fmt` which is a part of `make test` it's strange it is not used in the upstream.